cradle-cassandra/src/main/java/com/exactpro/cradle/cassandra/CassandraStorageSettings.java

```diff
@@ -27,11 +27,12 @@
 import static com.exactpro.cradle.CradleStorage.DEFAULT_MAX_MESSAGE_BATCH_SIZE;
 import static com.exactpro.cradle.CradleStorage.DEFAULT_MAX_TEST_EVENT_BATCH_SIZE;
 
+@SuppressWarnings("unused")
 @JsonIgnoreProperties(ignoreUnknown = true)
-public class CassandraStorageSettings {
+public class CassandraStorageSettings extends CoreStorageSettings {
     public static final String SCHEMA_VERSION = "5.3.0";
 
-    public static final ConsistencyLevel DEFAULT_CONSISTENCY_LEVEL = ConsistencyLevel.LOCAL_QUORUM;
+    public static final CassandraConsistencyLevel DEFAULT_CONSISTENCY_LEVEL = CassandraConsistencyLevel.LOCAL_QUORUM;

```

Comment:
Why do we use a different class here?

Reply:
CassandraConsistencyLevel is our class and it can be provided as API instead of ConsistencyLevel

Reply:
But I believe that we already have this change in dev-version-5?...
https://github.com/th2-net/cradleapi/blob/dev-version-5/cradle-cassandra/src/main/java/com/exactpro/cradle/cassandra/CassandraStorageSettings.java#L30

Reply:
Merged with dev-version-5